oss/oss.go

```diff
@@ -0,0 +1,779 @@
+				Help:  "east America (Virginia) Region.",
+			}, {
+				Value: "ap-southeast-1",
+				Help:  "Asia Pacific (Singapore) Region.",
+			}, {
+				Value: "ap-southeast-2",
+				Help:  "Asia Pacific (Sydney) Region.",
+			}, {
+				Value: "ap-northeast-1",
+				Help:  "Asia Pacific (Japanese).",
+			}, {
+				Value: "eu-central-1",
+				Help:  "Central Europe (Frankfurt).",
+			}, {
+				Value: "me-east-1",
+				Help:  "Middle East (Dubai) Region.",
+			}},
+		}, {
+			Name: "acl",
+			Help: "Canned ACL used when creating buckets and/or storing objects in oss.",
+			Examples: []fs.OptionExample{{
+				Value: "private",
+				Help:  "Owner gets FULL_CONTROL. No one else has access rights (default).",
+			}, {
+				Value: "public-read",
+				Help:  "Owner gets FULL_CONTROL. The AllUsers group gets READ access.",
+			}, {
+				Value: "public-read-write",
+				Help:  "Owner gets FULL_CONTROL. The AllUsers group gets READ and WRITE access.\nGranting this on a bucket is generally not recommended.",
+			}, {
+				Value: "default",
+				Help:  "Owner gets FULL_CONTROL.the bucket is not recommended.",
+			}},
+		}, {
+			Name: "storage_class",
+			Help: "StorageClassType Bucket的存储类型",
+			Examples: []fs.OptionExample{{
+				Value: "Standard",
+				Help:  "StorageStandard 标准存储模式",
+			}, {
+				Value: "Archive",
+				Help:  "StorageArchive 归档存储模式",
+			}, {
+				Value: "IA",
+				Help:  "StorageIA 低频存储模式",
+			}},
+		}},
+	})
+}
+
+// Constants
+const (
+	metaMtime      = "Mtime"                // the meta key to store mtime in - eg X-Amz-Meta-Mtime
+	listChunkSize  = 1000                   // number of items to read at once
+	maxRetries     = 10                     // number of retries to make of operations
+	maxSizeForCopy = 5 * 1024 * 1024 * 1024 // The maximum size of object we can COPY
+)
+
+// Globals
+var (
+	ossACL          = fs.StringP("oss-acl", "", "", "Canned ACL used when creating buckets and/or storing objects in OSS")
+	ossStorageClass = fs.StringP("oss-storage-class", "", "", "Storage class to use when uploading OSS objects (Standard|Archive|IA)")
+)
+
+// Fs represents a remote oss server
+type Fs struct {
+	name               string       // the name of the remote
+	root               string       // root of the bucket - ignore all objects above this
+	features           *fs.Features // optional features
+	c                  *oss.Client  // the connection to the oss server
+	bucket             string       // the bucket we are working on
+	bucketOKMu         sync.Mutex   // mutex to protect bucket OK
+	bucketOK           bool         // true if we have created the bucket
+	locationConstraint string       // location constraint of new buckets
+	storageClass       string       // storage class
+	acl                string       // bucket acl
+}
+
+// Object describes a oss object
+type Object struct {
+	// Will definitely have everything but meta which may be nil
+	// List will read everything but meta & mimeType - to fill
```

Comment:
Correct comment. I don`t know what you mean. It`s not written in English.
Do the same for comment below.